src/ccl_error.c

```diff
@@ -69,11 +69,16 @@ void ccl_check_status_nocosmo(int * status)
     fprintf(stderr,"%s", "CCL_ERROR_HMF_INTERP");
     exit(1);
```

Comment:
I'm worried that these `exit()` statements could crash the Python wrapper. I will look into this (it's not your fault though).

Reply:
I fixed this in a different PR that's now been merged with `master`.